tv_apps/smart-home/js/edit.js

```diff
@@ -65,67 +63,78 @@
       this.isFolderReady = false;
       // The hovering card
       this._hoveringCard = null;
-      this.mainSection.dataset.mode = '';
+      this.mode = '';
```

Comment:
Let's use `this._home.mode` directly and remove the getter and setter of `Edit` object.